install-deps.sh

```diff
@@ -212,7 +212,9 @@ if [ x$(uname)x = xFreeBSDx ]; then
         java/junit \
         lang/python \
         lang/python27 \
+        lang/python36 \
         devel/py-pip \
+	devel/py-flake8 \
```

Comment:
could you use spaces for indent? just to be consistent with other packages in this command.

Reply:
@tchaikov 
Fix other ommissions as well